src/Components/ArticleCard.jsx

```diff
@@ -1,4 +1,27 @@
-export default function ArticleCard({ article }) {
+import axios from "axios";
+import { useState } from "react";
+
+export default function ArticleCard({ article, setArticleList }) {
+
+  function handleLike(e) {
+    e.preventDefault();
+    setArticleList((currList) => {
+      const newList = currList.map((item) => {
+        if (item.article_id === +e.target.value) {
+          item.votes = item.votes + +e.target.id;
+        }
+        return item;
+      });
+      return newList;
+    });
+    const body = {
+      inc_votes: e.target.id,
+    };
+    axios.patch(
+      `https://northcoders-news-03ck.onrender.com/api/articles/${e.target.value}`,
```

Comment:
We could have a file where we use axios create and a function for each endpoint, which means in the component you would only need to invoke the the relevant function that will have a more descriptive name 🙂